nucypher/characters/lawful.py

```diff
@@ -1466,11 +1469,26 @@ def encrypt_for_pre(
         )
         return message_kit
 
-    def encrypt_for_dkg(self, plaintext: bytes, conditions: Lingo) -> Ciphertext:
+    def encrypt_for_dkg(self, plaintext: bytes, conditions: Lingo) -> DkgMessageKit:
```

Comment:
I also want to take a pause here to build on my above comment and revisit the discussion about keeping the developer/user-facing abstraction `Ciphertext`.  Perhaps we don't need a new name here, just a new format.